yksgi/src/llvmbridge.rs

```diff
@@ -188,6 +188,18 @@ impl Value {
         unsafe { !LLVMIsASwitchInst(self.0).is_null() }
     }
 
+    pub fn is_inline_asm(&self) -> bool {
+        !unsafe { LLVMIsAInlineAsm(self.0).is_null() }
+    }
+
+    pub fn is_function(&self) -> bool {
+        unsafe { !LLVMIsAFunction(self.0).is_null() }
+    }
+
+    pub fn is_vararg_function(&self) -> bool {
+        unsafe { LLVMIsFunctionVarArg(LLVMGetElementType(LLVMTypeOf(self.0))) != 0 }
```

Comment:
s/LLVMTypeOf(self.0)/self.get_type()

Reply:
I considered this. It would have to be `self.get_type().get()`, at which point I thought "meh".

Reply:
Maybe a good candidate to implement `LLVMGetElementType`. I think we are using it in other places too. So we get maybe `self.get_type().get_element()`?

Reply:
Actually that wrapper already exists, just we don't use it.

Fixed in af5b4fc5df (to be squashed in) and cce4e3a3c (to be left separate).